### PR TITLE
Use open source images in tests

### PR DESCRIPTION
**Pull request by connorgorman**

### CODE REVIEW

**Review by RTann — APPROVED** [review]
going to approve to avoid another round, but you'll need to remove the `gcr.io` references in the Makefile, too

File: Makefile [hunk 1]
```diff
@@ -219,14 +219,14 @@ deploy: clean-helm-rendered
 slim-deploy: clean-helm-rendered
 	@echo "+ $@"
 	kubectl create namespace stackrox || true
-	helm template scanner chart/ --set scannerImage=us.gcr.io/stackrox-ci/scanner-slim,scannerDBImage=us.gcr.io/stackrox-ci/scanner-db-slim,tag=$(TAG),logLevel=$(LOGLEVEL),updateInterval=2m --output-dir rendered-chart
+	helm template scanner chart/ --set scannerImage=quay.io/stackrox-io/scanner-slim,scannerDBImage=quay.io/stackrox-io/scanner-db-slim,tag=$(TAG),logLevel=$(LOGLEVEL),updateInterval=2m --output-dir rendered-chart
 	kubectl apply -R -f rendered-chart
 
 .PHONY: deploy-dockerhub
 deploy-dockerhub: clean-helm-rendered
```
Comment (RTann) [inline Makefile:226]: just noticed this. Do you recall what this is for @connorgorman ?

Comment (connorgorman) [reply Makefile:226]: oh deploying locally

Comment (RTann) [reply Makefile:226]: 👍 seems fine to keep then

Comment (RTann) [reply Makefile:226]: maybe revert the change here, though since `make image` will build `stackrox/scanner`

Comment (connorgorman) [reply Makefile:226]: yeah good point

Comment (connorgorman) [reply Makefile:226]: well on a quick second thought, you wouldn't be able to deploy CI cause there is no dockerhub. Should just tag with OS then

Comment (RTann) [reply Makefile:226]: CI uses `make deploy` or `make deploy-slim` which you tag properly

Comment (RTann) [reply Makefile:226]: maybe we rename this to `deploy-local`?

**Comment by RTann** [comment]
Moving to another PR. Don't want to deal with the rebase

**Pull request closed without merging**